### PR TITLE
Update naviagation

### PR DESCRIPTION
**Pull request by burakukula**
Screenshot 2020-01-15 at 18 33 46
Screenshot 2020-01-15 at 18 33 53
Screenshot 2020-01-15 at 18 34 00
Screenshot 2020-01-15 at 18 34 12
Screenshot 2020-01-15 at 18 34 20

Screenshot 2020-01-15 at 18 34 34
Screenshot 2020-01-15 at 18 34 41
Screenshot 2020-01-15 at 18 34 50

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     
     <link rel="icon" href="/images/icon.png">
-    <link rel="apple-touch-icon" href="/images/icon.png">
+    <link rel="apple-touch-icon" href="/images/icon-bg.png">
```
Comment (burakukula) [inline src/index.html:20]: Let's check if this one is working, it's a png with white bg, I would like to test it.

File: scripts/deploy.js [hunk 2]
```diff
@@ -1,24 +1,26 @@
...
 
 const config = {
   user: loginData.user,
   password: loginData.password,
-  host: "ftp.cluster028.hosting.ovh.net",
+  host: 'ftp.cluster028.hosting.ovh.net',
   port: 21,
   localRoot: path.join(__dirname, '../dist'),
-  remoteRoot: '/www',
+  remoteRoot: '/',
```
Comment (vergilius) [inline scripts/deploy.js:14]: Hm?

**Review by vergilius — APPROVED** [review]

File: scripts/deploy.js (hunk 2 above)
Comment (burakukula) [reply scripts/deploy.js:14]: we supposed to remove it? or not?

Comment (burakukula) [reply scripts/deploy.js:14]: I am running deploy right now and it seems like it's working. 😉(so yeah, I guess it should be uploaded to `'/'`)

**Pull request merged**